packages/node_modules/nav-frontend-alertstriper/src/alertstripe.tsx

```diff
@@ -47,6 +47,10 @@ export interface AlertStripeProps {
      * -
      */
     className?: string;
+    /**
+     * -
+     */
+    role?: string;
```

Comment:
Hm, trenger vi egentlig å definere denne propen? `role` er vel et gyldig HTML-attributt på `<div>`-elementer 🤔I tilfellet Typescript-kompilatoren klager; kanskje interfacet bare kan extende `HTMLAttributes<HTMLDivElement>`?

Reply:
Vi ekstender ikke html-props noe sted, gjør vi det

Reply:
Hm, nei - lurer bare på om det er unødvendig å definere `role` på interfacet. Usikker på om vi trenger å extende noe for å unngå at folk får kjeft av ts-kompilatoren sin. Ser vi gjør det her f.eks.: https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-knapper/src/CustomHTMLButtonAttributes.ts

Er det mulig å bare fjerne role fra interface uten å extende noe?